spec/cypress/integration/lead_providers/ReportSchools.feature

```diff
@@ -10,3 +10,14 @@ Feature: Report Schools flow
     And "page body" should contain "2021"
     And the page should be accessible
     And percy should be sent snapshot called "Lead provider report schools start page"
+
+  Scenario: Selecting a delivery partner
+    Given scenario "lead_provider_with_delivery_partners" has been run
```

Comment:
I think that's something I would put in `Background` part - hopefully all tests in this spec will need a setup like that.